File: UnicoVehicle/UnicoVehicle/UnicoVehicle.DTO/InsuranceCompany.cs
using System;
namespace UnicoVehicle.DTO
{
    public class InsuranceCompany
    {
        public int InsuranceCompanyId { get; set; }
        public Miscellaneous.District District { get; set; }
        public string InsuranceCompanyName { get; set; }
        public string Address { get; set; }
        public string CountryHead { get; set; }
    }
}

File: Flutter App/unicoVehicle/lib/Models/InsuranceCompany.dart
import './Miscellaneous/District.dart';

class InsuranceCompany {
  final int insuranceCompanyId;
  final District district;
  final String address;
  final String countryHead;
  final String companyName;

  InsuranceCompany({
    this.address,
    this.companyName,
    this.countryHead,
    this.district,
    this.insuranceCompanyId,
  });
}
